templates/terraform/examples/ml_model_basic.tf.erb

```diff
@@ -0,0 +1,5 @@
+resource "google_ml_engine_model" "default" {
+  name = "default"
```

Comment:
This name should be random, that way in case it doesn't get cleaned up on a test run, the next run won't collide. I think most of our examples already do this so it should be pretty easy to find something to base it off of

Reply:
Fixed with the other PR!